### PR TITLE
Cat ears are now 200% more cursed

### PR DESCRIPTION
**Pull request by jknpj**
HEWWO EVEWYONYE!

:cl:
 * rscadd: Cursed cat ears are now 200% worse than before.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
wearing cat ears giving you brain damage again when

**Comment by AI-Is-Rogue** [comment]
never ever

**Comment by boy2mantwicethefam** [comment]
Why

**Comment by Kharkov72** [comment]
🤔

**Comment by BomberBro** [comment]
It's content.

**Comment by AI-Is-Rogue** [comment]
>why


Because that wasn't locked behind an esoteric mechanic that will only be seen 1/15 rounds 😏 
Really though it's because grudgecoding bad

**Comment by Kammerjunk** [comment]
floor length braid tripping you is the single best feature in the /vg/station codebase
we should have more features like it

**Comment by AI-Is-Rogue** [comment]
agree on floor length braid disagree on turning existing items into items that inflict babality on you

**Comment by SonixApache** [comment]
good content

**Comment by boy2mantwicethefam** [comment]
@AI-Is-Rogue Cat ears giving brain damage is just representative of the real thing

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -182,15 +182,17 @@
 	desc = "A pair of kitty ears. Meow!"
 	icon_state = "kitty"
 	flags = FPRINT
-	var/haircolored = 1
+	var/haircolored = TRUE
+	var/cringe = FALSE
```
Comment (ShiftyRail) [inline code/modules/clothing/head/misc_special.dm:186]: good var name

**Comment by AI-Is-Rogue** [comment]
@boy2mantwicethefam it all makes sense now

**Pull request merged**